### ISSUE TITLE
Server meter not working in master

### ISSUE THREAD

**Issue by miczac** [supercollider/supercollider#2067]
Meter GUI shows up but indicators don't work.
Works in 3.7.1 and until [58a859acffbab27870e716d44659b18d2c74b9c0]
```
(
s.waitForBoot({
s.meter;
{SinOsc.ar([69,76.15].midicps,mul:[-20,-30].dbamp)}.play // just a test signal ;)
}))
```
This doesn't work anymore with commit 80eddc8    -> #1935  ?

**Comment by telephon** [comment]
fixed in https://github.com/supercollider/supercollider/pull/2066

**Comment by miczac** [comment]
just checked: works after applying https://github.com/supercollider/supercollider/pull/2066
Thanks!

**Comment by miczac** [comment]
whoops, wrong kind of close?

**Comment by miczac** [comment]
fixed via #2066
